packages/mobile/src/merchantPayment/FeeContainer.test.tsx

```diff
@@ -0,0 +1,18 @@
+import { render } from '@testing-library/react-native'
+import BigNumber from 'bignumber.js'
+import * as React from 'react'
+import 'react-native'
+import { Provider } from 'react-redux'
+import FeeContainer from 'src/merchantPayment/FeeContainer'
+import { createMockStore } from 'test/utils'
+
+describe('FeeContainer', () => {
+  it('renders correctly', () => {
+    const tree = render(
+      <Provider store={createMockStore({})}>
+        <FeeContainer amount={new BigNumber(1337)} />
+      </Provider>
+    )
+    expect(tree).toMatchSnapshot()
```

Comment:
This only check that it renders, but it's not actually useful.
We've decided to move away from this kind of test and favor of https://testing-library.com/ tests.

Reply:
This comment is also for the other component tests that were added.